octobot_evaluators/evaluator/abstract_evaluator.py

```diff
@@ -248,21 +241,26 @@ def set_eval_note(self, new_eval_note) -> None:
             self.eval_note += new_eval_note
 
     @classmethod
-    def is_enabled(cls, config, default) -> dict:
+    def is_enabled(cls, tentacles_setup_config, default) -> bool:
         """
         Check if the evaluator is enabled by configuration
-        :param config: global config
+        :param tentacles_setup_config: tentacles setup config
         :param default: default value if evaluator config is not found
         :return: evaluator config
         """
-        if config[CONFIG_EVALUATOR] is not None:
-            if cls.get_name() in config[CONFIG_EVALUATOR]:
-                return config[CONFIG_EVALUATOR][cls.get_name()]
-            else:
-                for parent in cls.mro():
-                    if parent.__name__ in config[CONFIG_EVALUATOR]:
-                        return config[CONFIG_EVALUATOR][parent.__name__]
-                return default
+        try:
+            return is_tentacle_activated_in_tentacles_setup_config(tentacles_setup_config,
+                                                                   cls.get_name(),
+                                                                   raise_errors=True)
+        except KeyError:
+            for parent in cls.mro():
```

Comment:
Why are we checking if the tentacle parent is enabled ?

Reply:
For advanced tentacles, you might have an advanced daily trading mode that is enabled because daily trading mode is enabled

Reply:
Oh ok I didn't remember this behavior